source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -561,8 +561,12 @@ def _get_description(self):
 				# Translators: Indicates that there is 1 series in a chart.
 				seriesValueString = _( "There is 1 series in this chart" )
 			else:
-				# Translators: Indicates the number of series in a chart where there are multiple series.
-				seriesValueString = _( "There are total %d series in this chart" ) %(count)
+				seriesValueString = ngettext(
+					# Translators: Indicates the number of series in a chart where there are multiple series.
+					"There is total %d series in this chart",
+					"There are total %d series in this chart",
```

Comment:
can we fix the grammar here? 

```suggestion
					"There is %d total series in this chart",
					"There are %d total series in this chart",
```